src/views/auth/Sidebar.vue

```diff
@@ -9,16 +9,18 @@
       <strong>{{ companyName }}</strong>
     </div>
     <nav class="sidebar__nav">
-      <router-link
-        :to="{ name: 'Dashboard' }"
-      >
-        Dashboard
-      </router-link>
-      <!-- <router-link
-        :to="{ name: 'CompanySettings', params: { id: userUid } }"
-      >
-        Settings
-      </router-link> -->
+      <slot>
```

Comment:
what is the reason for using `slot` here?

Reply:
So we can customize it if needed